flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/source/AbstractTableInputFormat.java

```diff
@@ -99,6 +108,14 @@
 	 */
 	public abstract void configure(Configuration parameters);
 
+	protected void setSerializedConfig(org.apache.hadoop.conf.Configuration hConf) {
+		serializedConfig = HBaseConfigurationUtil.serializeConfiguration(hConf);
+	}
+
+	protected org.apache.hadoop.conf.Configuration getSerializedConfig() {
```

Comment:
How about to call it `getHadoopConfiguration` ? Because the returned value is not "serialized".